### ISSUE TITLE
Support DTED

### ISSUE THREAD

**Issue by cglusky** [goodrobots/maverick-web#26]
Digital Terrain Elevation Data.

May go by other names in other places.

Sourcing good data can be an issue.  Need to investigate latest.

**Comment by fnoop** [comment]
I'm assuming this is taken care of inherently by the cesium 3d terrain sets?

**Comment by cglusky** [comment]
Not sure.  There was some work done to ry to standardize things like buildings.  Lost track.

https://osmbuildings.org/

Might need to abstract the title of the issue once i get more info.

**Comment by fnoop** [comment]
This is like organised squirreling.

**Comment by cglusky** [comment]
yep

**Comment by cglusky** [comment]
@fnoop yes, it should be part of the data set used by cesium as far as i can tell.  which i guess means we just need a way to query elevation data for whatever time and space you need.

https://www.citygml.org/

path planning and safety checks on min/max altitude etc would be the use case.

**Comment by cglusky** [comment]
This could have a big impact on speed of adoption by removing some barriers.  it can be hard to generate good 3D maps.

https://cesium.com/blog/2017/01/24/cesium.com-update/

**Comment by SamuelDudley** [comment]
@cglusky I have a cesium.com beta account. The new thing for rendering cities and other details is 3D tiles. http://www.aero3dpro.com.au/adelaide_3D_city_model_2017

**Comment by brizey02** [comment]
Cesium has its own terrain data, You can also compile your own for your own use, its a bit of a pain though. The buildings and such should be other 3D tiles not related to terrain.
